starlite/cli/commands/core.py

```diff
@@ -52,7 +66,7 @@ def run_command(
     """
 
     try:
-        import uvicorn
+        pass
```

Comment:
empty try?

Reply:
Well, that can be removed :upside_down_face: